ui/zenoplayer/zenoplayer.cpp

```diff
@@ -14,9 +14,12 @@ ZenoPlayer::ZenoPlayer(ZENO_PLAYER_INIT_PARAM param, QWidget *parent)
     : QWidget(parent), m_InitParam(param) 
 {
     setObjectName("ZenoPlayer");
-    resize(1000, 680);
-    setMinimumSize(1000, 680);
+//    resize(1000, 680);
+//    setMinimumSize(1000, 680);
+    setFixedSize(1080, 761);  // 720 + 41(menus res)
```

Comment:
```suggestion
    resize(1000, 680);
    setMinimumSize(1000, 680);
```
不是所有平台的menu，都刚好和你的ubuntu一样大。